docs/docusaurus/Dockerfile

```diff
@@ -2,11 +2,7 @@ FROM node:8.11.4
 
 WORKDIR /app/website
 
-EXPOSE 3000 35729
-COPY docusaurus/package.json /app/website/package.json
+COPY package.json /app/website/package.json
 RUN yarn install
-COPY docusaurus /app/website
-COPY readmes /app/docs
-RUN yarn build
 
-CMD ["yarn", "start"]
+CMD yarn install && yarn start
```

Comment:
Should we add `yarn install` in CMD? The container will take more tie to start then.

Reply:
The install directory is a volume mounted to the host fs, so total startup time should be similar to before

I also considered whether it would be worthwhile even if it was slower, and I think it would indeed be worth it for the benefit of instant live reload. But thankfully we get best of both worlds